aws/rds_test.go

```diff
@@ -0,0 +1,92 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	"strings"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+func waitUntilRdsCreated(svc *rds.RDS, name *string) error {
```

Comment:
Like the comment above about `waitUntilRdsDeleted`, possibly replaced with [WaitUntilDBInstanceAvailable](https://docs.aws.amazon.com/sdk-for-go/api/service/rds/#RDS.WaitUntilDBInstanceAvailable).

Reply:
I don't know why but using this function it just never ends :( I'll leave with the custom one.

Reply:
Ok sounds good! Can you add a comment indicating that you've tried to use that function, but ran into mysterious infinite loops? It would help set the context in the future when we come back here and someone has the same thought as me.